<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alyaWidget</class>
 <widget class="QStackedWidget" name="alyaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>780</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="subMain">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="alyaProblemTabs">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <<alyaTabs>>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="modulesServicesButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>100</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Modules and Services</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runSimulationButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>100</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Run simulation</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="modulesServices">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>760</height>
        </rect>
       </property>
       <widget class="QLineEdit" name="problemNameEdit">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>60</y>
          <width>141</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="alyaServices">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>310</y>
          <width>256</width>
          <height>111</height>
         </rect>
        </property>
        <<alyaServices>>
       </widget>
       <widget class="QLabel" name="l_problemservices_2">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>290</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Problem services</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addService">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>330</y>
          <width>61</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>&gt;&gt;&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="l_alyaservices">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>290</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Alya services</string>
        </property>
       </widget>
       <widget class="QPushButton" name="continueButton">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>440</y>
          <width>97</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Continue</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_34">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Problem name:</string>
        </property>
       </widget>
       <widget class="QListWidget" name="problemServices">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>310</y>
          <width>256</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>140</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Problem modules</string>
        </property>
       </widget>
       <widget class="QPushButton" name="delService">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>380</y>
          <width>61</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Alya modules</string>
        </property>
       </widget>
       <widget class="QListWidget" name="problemModules">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>160</y>
          <width>256</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="delModule">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>220</y>
          <width>61</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>381</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Century Schoolbook L</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Problem modules and services</string>
        </property>
       </widget>
       <widget class="QPushButton" name="editDomainButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>100</y>
          <width>97</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addModule">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>170</y>
          <width>61</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>&gt;&gt;&gt;</string>
        </property>
       </widget>
       <widget class="QListWidget" name="alyaModules">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>256</width>
          <height>111</height>
         </rect>
        </property>
        <<alyaModules>>
       </widget>
       <widget class="QLabel" name="label_35">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Problem domain:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="importDomainButton">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>100</y>
          <width>97</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="domainDefinedCheck">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>100</y>
          <width>97</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Defined</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="runPage">
   <widget class="QDockWidget" name="dockWidget_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>421</width>
      <height>171</height>
     </rect>
    </property>
    <property name="features">
     <set>QDockWidget::NoDockWidgetFeatures</set>
    </property>
    <property name="windowTitle">
     <string>Parameters</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_4">
     <widget class="QWidget" name="formLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_10">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Connection:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="alyaConnectionCombo">
         <item>
          <property name="text">
           <string>default</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>With MPI:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="withMPICheck">
         <property name="text">
          <string>yes/no</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>Processes number:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="procNumberEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="runButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>97</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Run!</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QDockWidget" name="dockWidget_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>411</width>
      <height>351</height>
     </rect>
    </property>
    <property name="features">
     <set>QDockWidget::NoDockWidgetFeatures</set>
    </property>
    <property name="windowTitle">
     <string>Simulation</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_5">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout_11">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>Progression</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>Output:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTextEdit" name="outputRunEdit">
           <property name="textInteractionFlags">
            <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="modulesServicesButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>610</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Modules and Services</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>610</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Problem definition</string>
    </property>
   </widget>
   <widget class="QDockWidget" name="resultsDock">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>411</width>
      <height>70</height>
     </rect>
    </property>
    <property name="features">
     <set>QDockWidget::NoDockWidgetFeatures</set>
    </property>
    <property name="windowTitle">
     <string>Results</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_6">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="openResultButton">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateResultButton">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
<<customWidgets>>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
